crates/curp-external-api/src/cmd.rs

```diff
@@ -215,3 +192,29 @@ impl From<DecodeError> for PbSerializeError {
         PbSerializeError::RpcDecode(err)
     }
 }
+
+/// After sync command type
+#[derive(Debug)]
+pub struct AfterSyncCmd<'a, C> {
+    /// The command
+    cmd: &'a C,
+    /// Whether the command needs to be executed in after sync stage
+    to_exectue: bool,
+}
+
+impl<'a, C> AfterSyncCmd<'a, C> {
+    /// Creates a new `AfterSyncCmd`
+    pub fn new(cmd: &'a C, to_exectue: bool) -> Self {
+        Self { cmd, to_exectue }
+    }
+
+    /// Gets the command
+    pub fn cmd(&self) -> &'a C {
+        self.cmd
+    }
+
+    /// Convert self into parts
+    pub fn into_parts(self) -> (&'a C, bool) {
```

Comment:
What does the `parts` mean here?

Reply:
`into_parts` means to decompose the struct into its constituent parts.